src/controls/extras/tree.coffee

```diff
@@ -170,6 +170,9 @@ class Tree extends Typed
           setTimeout(arguments.callee, 0)
 
     timer = false
+    bandsCountId = "bands-count-" + $('.bands-count').length
```

Comment:
I agree with Bo too. Just haven't made the change yet - the rspec rest has been keeping me entertained.  Will update soon.